proto/buf/alpha/registry/v1alpha1/reference.proto

```diff
@@ -72,3 +78,17 @@ message GetReferenceByNameRequest {
 message GetReferenceByNameResponse {
   Reference reference = 1;
 }
+
+message ListGitCommitMetadataForReferenceRequest {
+  // Name of the requested reference.
+  string name = 1;
+  // Owner of the repository the reference belongs to.
+  string owner = 2;
+  // Name of the repository the reference belongs to.
+  string repository_name = 3;
+}
+
+message ListGitCommitMetadataForReferenceResponse {
+  // List of git commits and metadata associated with the resolved reference.
+  repeated GitCommitMetadata git_commit_metadatas = 1;
+}
```

Comment:
I'd also return the BSR commit id or commit name (or both) that was resolved from the passed reference.

Reply:
Oh that's a good idea -- I think ID should be reasonable, since if we need to resolve additional information, the ID would be most useful.